packages/pipeline/src/utils/index.ts

```diff
@@ -49,3 +49,12 @@ class BigNumberTransformer implements ValueTransformer {
 }
 
 export const bigNumberTransformer = new BigNumberTransformer();
+
+/**
+ * Returns the unix timestamp of the current hour
+ */
+export function getHourInUnixTime(): number {
```

Comment:
This doesn't seem to be used anywhere. Should we remove it?

Reply:
I had already had a pretty significant amt of design done for CC price/volume scaping, this was part of that until priority shifted last week. Going to sync with xianny but 90% sure we'll need this function to interface with cryptocompare

Reply:
👍 okay with keeping it then.